index.html

```diff
@@ -42,5 +42,119 @@
 
     </header>
 
-  </body>
+    <main class="main-index">
+      <h1 class="visually-hidden">Интернет-магазин device</h1>
+      <img class="logo" src="#" alt="квадрокоптер на желто-белом фоне">
+    </main>
+    <section class="advantages">
+      <h2 class="visually-hidden">Преимущества</h2>
+      <span>Порхает как бабочка, жалит как пчела!</span>
```

Comment:
```suggestion
      <p>Порхает как бабочка, жалит как пчела!</p>
```
Это ведь целая фраза, а не кусок строки